package.json

```diff
@@ -43,6 +43,7 @@
     "rollback": "./development/rollback.sh"
   },
   "dependencies": {
+    "3box": "^1.8.0",
```

Comment:
Looking forward to moving plugins into their own repos, will really neatly prove that we aren't even requiring our devs to install these "potentially untrusted dependencies".